solidity/contracts/RepCrowdsourcer.sol

```diff
@@ -24,13 +19,13 @@ contract GoFundMicah {
 	function deposit(uint256 amount) public {
 		require(!contractClosed, 'Deposits are closed');
 		repV2.transferFrom(msg.sender, address(this), amount);
-		deposits[msg.sender] += amount;
+		_mint(msg.sender, amount);
 		emit Deposit(msg.sender, amount);
 	}
 
 	function internalWithdraw(address recipient) private {
-		uint256 amount = deposits[recipient];
-		deposits[recipient] = 0;
+		uint256 amount = _balances[recipient];
+		_burn(recipient, amount);
```

Comment:
Any reason not to inline this?
```suggestion
		_burn(recipient, _balances[recipient];);
```

Reply:
amount is referenced twice elsewhere in the function so we'd be doing additional storage lookups for no reason is why it's not inlined. Will work on the whole Micah returning partial amount thing in this PR too!